dynatrace-oneagent-operator/README.md

```diff
@@ -0,0 +1,93 @@
+# Dynatrace OneAgent Operator Helm Chart
+
+This is the home of Dynatrace OneAgent Operator's Helm Chart which supports the rollout and lifecycle of [Dynatrace OneAgent](https://www.dynatrace.com/support/help/get-started/introduction/what-is-oneagent/) in Kubernetes and OpenShift clusters.
+Rolling out Dynatrace OneAgent via DaemonSet on a cluster is straightforward.
+Maintaining its lifecycle places a burden on the operational team.
+Dynatrace OneAgent Operator closes this gap by automating the repetitive steps involved in keeping Dynatrace OneAgent at its latest desired version.
+
+## Overview
```

Comment:
Would suggest to also include that the Helm chart *WILL* create an OneAgent object automatically.

Reply:
Added